packages/addon/lua/gelly/mods/blood-mod/init.lua

```diff
@@ -31,51 +51,93 @@ local BLOOD_CONFIGS = {
 	},
 
 	-- shotgun
-	[536870914] = {
+	weapon_shotgun = {
 		MinDensity = 100,
 		MaxDensity = 200,
-		VelocityPower = 14,
-		Randomness = 0.1,
-		CubeSize = 5,
+		VelocityPower = 5,
+		Randomness = 0.5,
+		CubeSize = 15,
+		DamageMultiplier = 35,
 	},
 
-	[DMG_SLASH] = {
+	weapon_357 = {
 		MinDensity = 100,
 		MaxDensity = 200,
-		VelocityPower = 4,
+		VelocityPower = 5,
 		Randomness = 0.1,
-		CubeSize = 3,
+		CubeSize = 5,
+		DamageMultiplier = 30,
 	},
+}
 
-	[DMG_CLUB] = {
-		MinDensity = 100,
-		MaxDensity = 200,
-		VelocityPower = 4,
-		Randomness = 0.1,
-		CubeSize = 3,
+local BLOOD_COLOR_MATERIALS = {
+	[BLOOD_COLOR_RED] = {
+		Roughness = 0,
+		IsSpecularTransmission = true,
+		RefractiveIndex = 1.373,
+		Absorption = Vector(0.05, 0.5, 0.5),
+		DiffuseColor = Vector(0, 0, 0),
 	},
-
-	[DMG_BLAST] = {
-		MinDensity = 3200,
-		MaxDensity = 5200,
-		VelocityPower = 40,
-		Randomness = 1,
-		CubeSize = 10,
-		DamageMultiplier = 90,
+	[BLOOD_COLOR_YELLOW] = {
+		Roughness = 0,
+		IsSpecularTransmission = true,
+		RefractiveIndex = 1.373,
+		Absorption = Vector(0.25, 0.3, 0.5),
+		DiffuseColor = Vector(0, 0, 0),
+	},
+	[BLOOD_COLOR_GREEN] = {
+		Roughness = 0,
+		IsSpecularTransmission = true,
+		RefractiveIndex = 1.373,
+		Absorption = Vector(0.3, 0.25, 0.5),
+		DiffuseColor = Vector(0, 0, 0),
+	},
+	[BLOOD_COLOR_ANTLION] = {
+		Roughness = 0,
+		IsSpecularTransmission = true,
+		RefractiveIndex = 1.373,
+		Absorption = Vector(0.45, 0.1, 0.5),
+		DiffuseColor = Vector(0, 0, 0),
+	},
+	[BLOOD_COLOR_ZOMBIE] = {
+		Roughness = 0,
+		IsSpecularTransmission = true,
+		RefractiveIndex = 1.373,
+		Absorption = Vector(0.2, 0.35, 0.5),
+		DiffuseColor = Vector(0, 0, 0),
+	},
+	[BLOOD_COLOR_ANTLION_WORKER] = {
+		Roughness = 0,
+		IsSpecularTransmission = true,
+		RefractiveIndex = 1.373,
+		Absorption = Vector(0.5, 0.05, 0.5),
+		DiffuseColor = Vector(0, 0, 0),
 	},
 }
 
-local function sprayBlood(damageType, attacker, position, force, damage)
-	local normal = force:GetNormalized()
-	if attacker == LocalPlayer() then
-		normal = LocalPlayer():GetAimVector()
+local function getConfig(damageType)
```

Comment:
nit: `getDamageTypeConfig` to differentiate from the weapon config function below